### PR TITLE
fix: Update SDK, redact errors

### PR DESCRIPTION
**Pull request by disq**

### CODE REVIEW

File: client/error.go [hunk 1]
```diff
@@ -64,9 +63,7 @@ func ErrorClassifier(meta schema.ClientMeta, resourceName string, err error) []d
 	if ok := errors.As(err, &gerr); ok {
 		if grpcCode, ok := httpCodeToGRPCCode[gerr.Code]; ok {
 			if v, ok := grpcCodeToDiag[grpcCode]; ok {
-				return []diag.Diagnostic{
-					diag.FromError(err, v.severity, v.typ, resourceName, v.summary, err.Error()),
-				}
+				return execution.FromError(err, execution.WithSeverity(v.severity), execution.WithType(v.typ), execution.WithResource(resourceName), execution.WithSummary(v.summary), execution.WithDetails(err.Error()))
```
Comment (disq) [inline client/error.go:66]: Not sure about the `execution.WithDetails(err.Error())` part...

Comment (roneli) [reply client/error.go:66]: Yes, I agree, this might include user related info.

Comment (disq) [reply client/error.go]: Fixed

**Review by roneli — APPROVED** [review]
LGTM

**Review by roneli — APPROVED** [review]

**Pull request merged**